tests/baselines/reference/genericArrayWithoutTypeAnnotation.types

```diff
@@ -10,8 +10,8 @@ class Bar {
     public getBar(foo: IFoo[]) {
 >getBar : (foo: IFoo[]) => void
 >       : ^   ^^      ^^^^^^^^^
->foo : any[]
->    : ^^^^^
+>foo : IFoo<any>[]

```

Comment:
I feel like those kind of changes are especially nice here - when the error type appears as a type argument somewhere.